compiler/rustc_metadata/src/creader.rs

```diff
@@ -1172,7 +1172,7 @@ fn attempt_load_dylib(path: &Path) -> Result<libloading::Library, libloading::Er
         // the expected format is lib<name>.a(libname.so) for the actual
         // dynamic library
         let library_name = path.file_stem().expect("expect a library name");
-        let mut archive_member = OsString::from("a(");
+        let mut archive_member = std::ffi::OsString::from("a(");
```

Comment:
Question: is this qualification needed or a stray leftover change?

Reply:
Thanks for catching that, it is a stray left over change. Though we do require this to build as `std::ffi::OsString` is not imported and I made a mistake before since cfg macro gating the aix implementation. I can seperate this out into a different PR if needed.